packages/tradershub/src/screens/TermsOfUse/TermsOfUse.tsx

```diff
@@ -1,24 +1,36 @@
 import React from 'react';
 import { Form, Formik, FormikValues } from 'formik';
+import { Text } from '@deriv-com/ui';
 import Actions from '../../flows/RealAccountSIgnup/SignupWizard/Actions';
 import WizardScreenWrapper from '../../flows/RealAccountSIgnup/SignupWizard/WizardScreenWrapper';
 import { ACTION_TYPES, useSignupWizardContext } from '../../providers/SignupWizardProvider/SignupWizardContext';
+import FatcaDeclaration from './TermsOfUseSections/FatcaDeclaration';
+import PEPs from './TermsOfUseSections/PEPs';
+
+// Temporary till we have this in deriv-com/ui
+const Divider = () => {
+    return (
+        <div className='flex border-solid border-system-light-active-background bg-system-light-primary-background border-75' />
+    );
+};
```

Comment:
Not wrong but I feel there is a better way of writing this:
```suggestion
const Divider = () => (
        <hr className=' bg-system-light-primary-background' />
    );
```